vendor/decentlab/dl-iam-codec.yaml

```diff
@@ -7,8 +7,8 @@ uplinkDecoder:
         bytes: [0x02, 0x0b, 0xbd, 0x00, 0x7f, 0x0b, 0x92, 0x6a, 0x51, 0x5d, 0x48, 0xbc, 0x4e, 0x02, 0x62, 0x00, 0x69, 0x81, 0xc7, 0x00, 0x00, 0x93, 0xd4, 0x00, 0x0b, 0x01, 0x11]
       output:
         data:
-          device_id: 3005
           protocol_version: 2
+          device_id: 3005
```

Comment:
The order shouldn't matter; were you having issues?

Reply:
The order doesn't matter. It's just for future consistency. @jbiveroni